src/FixedDataTableCell.js

```diff
@@ -238,29 +226,24 @@ class FixedDataTableCell extends React.Component {
       }),
       props.className,
     );
-
-    var columnResizerComponent;
-    if (props.onColumnResize) {
-      var columnResizerStyle = {
+    var resizerComponent; // For backward compatibility
```

Comment:
Spawning the `ResizerKnob` when the user uses the older API.

Reply:
This means FDT is well aware of the resizer component. 
Is there any way to remove/minimize this coupling?